services/federation/follow_calls.go

```diff
@@ -0,0 +1,51 @@
+package federation
+
+import (
+	"github.com/yitsushi/go-misskey/core"
+	"github.com/yitsushi/go-misskey/models"
+)
+
+// FollowRequest is the request used by the followers and following commands.
+type FollowRequest struct {
+	Host    string `json:"host"`
+	SinceID string `json:"sinceId"`
+	UntilID string `json:"untilId"`
+	Limit   int    `json:"limit"`
+}
+
+// Follow is a single follower or following record.
```

Comment:
It should live under `models` as it's a data representation and can be used on other endpoints too (like #10). I'm not 100% sure about this, but my gut tells me...

Reply:
Verified: The `/users/followers` endpoint returns with the same object in an array (https://misskey.io/api-doc#operation/users/followers).